plugin-server/src/main/ingestion-queues/session-recording/session-recordings-consumer.ts

```diff
@@ -756,14 +766,37 @@ export class SessionRecordingIngester {
                 const highestOffsetToCommit = potentiallyBlockingOffset
                     ? potentiallyBlockingOffset - 1 // TRICKY: We want to commit the offset before the lowest blocking offset
                     : metrics.lastMessageOffset // Or the last message we have seen as it is no longer blocked
-                const lastKnownCommit = metrics.lastKnownCommit ?? -1
 
                 if (!highestOffsetToCommit) {
+                    if (partition === 101) {
+                        status.warn('🤔', 'blob_ingester_consumer - no highestOffsetToCommit for partition', {
+                            blockingSession: potentiallyBlockingSession?.sessionId,
+                            blockingSessionTeamId: potentiallyBlockingSession?.teamId,
+                            partition: partition,
+                            committedHighOffset,
+                            lastMessageOffset: metrics.lastMessageOffset,
+                            highestOffsetToCommit,
+                        })
+                    }
                     return
                 }
 
-                // If the last known commit is more than or equal to the highest offset we want to commit then we don't need to do anything
-                if (lastKnownCommit >= highestOffsetToCommit) {
+                // If the last known commit is ahead of the highest offset we want to commit then we don't need to do anything
+                if (committedHighOffset > highestOffsetToCommit) {
+                    if (partition === 101) {
```

Comment:
valuable enough that we don't need to clamp to 101?

Reply:
It is still super noisy. Typically we would get dozens of messages on every batch warning which is not really needed. Perhaps the one for "wants to commit lower than the current offset" might be worth keeping though...

Reply:
Actually no it will still log too much I think, as a blocking session will always have the same offset as the committed offset.